securedrop_client/gui/conversation/export/dialog.py

```diff
@@ -48,15 +48,11 @@ def __init__(self, device: Device, file_uuid: str, file_name: str) -> None:
 
         # Dialog content
         self.starting_header = _(
-            "Preparing to export:"
-            "<br />"
-            '<span style="font-weight:normal">{}</span>'.format(self.file_name)
-        )
+            "Preparing to export:<br />" '<span style="font-weight:normal">{}</span>'
+        ).format(self.file_name)
```

Comment:
The processing order makes sense. I would suggest using f-strings for translatable translations in order to make the string more self-contained and avoid making it depend on additional instructions where necessary.

```suggestion
        file_name = self.file_name
        self.starting_header = _(
            f'Preparing to export:<br /><span style="font-weight:normal">{file_name}</span>'
        )
```

I would also suggest we do 1. less HTML in our copy, and 2. expose it less to translators (something along the lines of `f"{start_bold}Preparing to export:{end_bold}{newline}{file_name}"`.) - but that's probably a step further than I'd go in the scope of this PR.

Reply:
@eloquence Pointed out to me that f-strings replacement happens immediately. That means that using an f-string in this case does exactly the opposite that this PR is trying to do: ensuring that placeholders are preserved as placeholders thorough the translation process, and replacement happens on the resulting translated strings. Indeed, my bad.

When we get to using _named placeholders_, [we can do it using `format()`](https://realpython.com/python-string-formatting/#2-new-style-string-formatting-strformat) as well:

```py
'Preparing to export:<br /><span style="font-weight:normal">{file_name}</span>'.format(file_name=self.file_name)

# or, to highlight the fact that we can send static strings for translation:

translatable_string = 'Preparing to export:<br /><span style="font-weight:normal">{file_name}</span>'  # this string is static, a.k.a "just a string"

translated_string = 'A preparar para exportar:<br /><span style=\"font-weight:normal\">{file_name}</span>'
translated_string.format(file_name=self.file_name)

# output:  'A preparar para exportar:<br /><span style=\"font-weight:normal\">memo.pdf</span>'
```

_Python's syntax highlighting looks pretty terrible, not sure what I'm doing wrong here..._

----

All that being said, @eloquence prefers not to actually modify the translatable string in the PR (to avoid triggering translation work, which we could handle ourselves in bulk), which makes sense. :+1: